README.md

```diff
@@ -312,7 +313,93 @@ Since their data is only base64 encoded, Kubernetes secrets should not be commit
   }
 ```
 
+### Deploying custom resources
 
+By default, kubernetes-deploy does not check to see whether the deployment of a custom resource is successful or not. Instead, it simply assumes that the custom resource deploys successfully and performs no additional checks. In order to meaningfully monitor the rollout of custom resources, kubernetes-deploy supports annotations that specify the pass/fail conditions of a given resource.
+
+>Note:
+This feature is only available on clusters running Kubernetes 1.11+ since it relies on the `metadata.generation` field being updated when custom resource specs are changed.
+
+*Requirements:*
+
+* The custom resource must expose a `status` field with an `observedGeneration` field.
+* The `kubernetes-deploy.shopify.io/cr-instance-rollout-conditions` annotation must be present on the *CRD* that defines the custom resource.
+* (optional) The `kubernetes-deploy.shopify.io/cr-instance-timeout` annotation can be added to the *CRD* that defines the custom resource to specify a default timeout for all instances of the CRD (otherwise the global default is used).
+
+### Specifying pass/fail conditions
+
+The presence of a valid `kubernetes-deploy.shopify.io/cr-instance-rollout-conditions` annotation on a CRD will cause kubernetes-deploy to monitor the rollout of any of its instances with the supplied annotation value. This value must be a valid JSON string with the following format:
```

Comment:
Does this need to be updated to say we also support `True`

Reply:
As this is an annotation, we can only support strings, not bools. Said that, kubernetes-deploy will downcase the value so things like `"True", "TRUE"`, etc. will work, though I don't think that needs to be documented